src/connection.js

```diff
@@ -426,4 +427,16 @@ export class StableWSConnection {
 			}
 		}, that.monitorInterval);
 	}
+
+	addClientVersion(url) {
+		const userAgent = `stream-chat-javascript-client-${
+			!this.browser ? 'node' : 'browser'
+		}-${pkg.version}`;
+		if (url.includes('?')) {
```

Comment:
I would rather move this inside `client.connect()` so you can skip the ugly `?` or `&` check and re-use the existing 
`_userAgent` method.

```js
client.wsURL = `${client.wsBaseURL}/connect?json=${qs}
    &api_key=${this.key}&authorization=${token}&stream-auth-type=${authType}
    &x-stream-client=${this._userAgent()}`;
```